docs/serving/installing-cert-manager.md

```diff
@@ -0,0 +1,121 @@
+environment:
+
+1. Run the following commands to download and extract the `cert-manager` installation package:
+
+    ```shell
+    CERT_MANAGER_VERSION=0.6.1
+    DOWNLOAD_URL=https://github.com/jetstack/cert-manager/archive/v${CERT_MANAGER_VERSION}.tar.gz
+
+    wget $DOWNLOAD_URL
+    tar xzf v${CERT_MANAGER_VERSION}.tar.gz
+
+    cd cert-manager-${CERT_MANAGER_VERSION} 
+    ```
+
+2. Depending on how you want to use cert-manager, choose one of the following
+   options for installing cert-manager:
+
+    - **Manually obtain and renew certificates**: Run the following command to 
+      install the cert-manager CRDs for Knative so that you can use the cert-manager tool for 
+      manually obtaining TLS certificates and adding them to your cluster.
+
+       ```shell
+       kubectl apply -f deploy/manifests/00-crds.yaml
+       ```
+
+    - **Enable Knative to automatically obtain and renew TLS certificates**: 
+    
+      1. Run one of the following commands to enable Knative to automatically obtain 
+         new TLS certificates and renew existing ones:
+      
+         - For Knative clusters running Kubernetes version 1.13 or above:
+
+           ```shell
+           # If you are running cluster in 1.13 or above
+           kubectl apply -f deploy/manifests/cert-manager.yaml 
+           ```
+
+         - For Knative clusters running Kubernetes version 1.12 or below:
+
+           ```shell
+           # If you are running cluster in 1.12 or below, you will need to add the --validate=false flag
+           kubectl apply -f deploy/manifests/cert-manager.yaml --validate=false
+           ```
+          
+         <!-- TODO(zhiminx) add the link of networking-certmanager deployment after the code is checked in.-->
+      1. If you install the Knative [networking-certmanager deployment]() 
+         in your cluster, you must also run the following command to install the Knative cert-manager CRDs: 
+          
+         ```shell
+         # If you are running cluster in 1.13 or above
+         kubectl apply -f deploy/manifests/cert-manager.yaml 
+         ```
+
+1. Configure which DNS provider is to validate the DNS-01 challenge requests.
+
+    By default, the [Let's Encrypt](https://letsencrypt.org) is used to
+    demonstrate how to configure cert-manager, but you can use other supported CA's 
+    that issue certificates with the ACME protocol. However, you
+    must use and configure your CA to use the
+    [`DNS-01` challenge type](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge).
+
+    Instructions about configuring cert-manager for any of the supported DNS providers are provided in
+    [DNS01 challenge providers and configuration instructions](https://docs.cert-manager.io/en/latest/tasks/acme/configuring-dns01/index.html#supported-dns01-providers).
+
+    **Example**
+
+    [Configure cert-manager for Google Cloud DNS](./using-auto-tls.md#Set-up-DNS-challenge-Provider)
```

Comment:
Open item: This Google Cloud DNS topic should probably convert back to a stand alone file. (see other comment)

Reply:
Done